### PR TITLE
Stephen's AMC testing script.

### PR DESCRIPTION
**Pull request by swh76**
## Issue
This PR doesn't resolve any issues but adds a script that Stephen wrote to qualify AMCs.

## Description

Adds new startup cfg file for qualifying AMCs, cfg_files/rflab/rflab_smurf_startup_stephen_test.cfg.  Adds _reg suffix to all smurf_atca_monitor register path variables.  Adds new function `get_amc_asset_tag` that returns an AMC's asset tag.  Adds Stephen's AMC testing script scratch/stephen/full_band_response_AMCatten.py.

## Does this PR break any interface?
- [ ] Yes
- [x] No

### CODE REVIEW

File: python/pysmurf/client/command/smurf_atca_monitor.py [hunk 1]
```diff
@@ -17,81 +17,125 @@
...
+        self._caput(
+            self.shelf_manager + self._write_atca_monitor_state_reg,
+            val, **kwargs)
 
-    def get_board_temp_fpga(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_fpga_reg = 'BoardTemp:FPGA'
 
+    def get_board_temp_fpga(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
```
Comment (eyyoung24) [inline python/pysmurf/client/command/smurf_atca_monitor.py:39]: should add doc strings here. Also, what is a typical `atca_epics_root`?

File: python/pysmurf/client/command/smurf_atca_monitor.py [hunk 2]
```diff
@@ -17,81 +17,125 @@
...
+            self._board_temp_fpga_reg,**kwargs)
 
-    _board_temp_rtm = 'BoardTemp:RTM'
-
-    def get_board_temp_rtm(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_rtm_reg = 'BoardTemp:RTM'
 
+    def get_board_temp_rtm(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
```
Comment (eyyoung24) [inline python/pysmurf/client/command/smurf_atca_monitor.py:53]: doc string

File: python/pysmurf/client/command/smurf_atca_monitor.py [hunk 3]
```diff
@@ -17,81 +17,125 @@
...
-    _board_temp_amc0 = 'BoardTemp:AMC0'
+        return self._caget(
+            f'{shelf_manager}:Crate:Slot_{slot_number}:' +
+            self._board_temp_rtm_reg,**kwargs)
 
-    def get_board_temp_amc0(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_amc0_reg = 'BoardTemp:AMC0'
 
+    def get_board_temp_amc0(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
```
Comment (eyyoung24) [inline python/pysmurf/client/command/smurf_atca_monitor.py:66]: These two functions should probably be writtten as `get_board_temp_amc` and then have an input var `amc_num`.

File: python/pysmurf/client/command/smurf_atca_monitor.py [hunk 4]
```diff
@@ -17,81 +17,125 @@
...
+        return self._caget(
+            f'{shelf_manager}:Crate:Slot_{slot_number}:' +
+            self._board_temp_amc2_reg,**kwargs)
 
-    def get_board_temp_amc2(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _junction_temp_fpga_reg = 'JunctionTemp:FPG'
 
+    def get_junction_temp_fpga(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
```
Comment (eyyoung24) [inline python/pysmurf/client/command/smurf_atca_monitor.py:95]: doc string. specifically, we need more words on what "junction" refers to here.

File: python/pysmurf/client/command/smurf_atca_monitor.py (hunk 2 above)
Comment (swh76) [reply python/pysmurf/client/command/smurf_atca_monitor.py:53]: This function not a part of this PR, but agree it needs a docstring.  Created issue to address https://github.com/slaclab/pysmurf/issues/497.

File: python/pysmurf/client/command/smurf_atca_monitor.py (hunk 3 above)
Comment (swh76) [reply python/pysmurf/client/command/smurf_atca_monitor.py:66]: This function not a part of this PR, but agree it needs a docstring.  Created issue to address https://github.com/slaclab/pysmurf/issues/497.

File: python/pysmurf/client/command/smurf_atca_monitor.py (hunk 4 above)
Comment (swh76) [reply python/pysmurf/client/command/smurf_atca_monitor.py:95]: This function not a part of this PR, but agree it needs a docstring.  Created issue to address https://github.com/slaclab/pysmurf/issues/497.

File: python/pysmurf/client/command/smurf_atca_monitor.py (hunk 1 above)
Comment (swh76) [reply python/pysmurf/client/command/smurf_atca_monitor.py:39]: This function not a part of this PR, but agree it needs a docstring.  Created issue to address https://github.com/slaclab/pysmurf/issues/497.

**Pull request merged**